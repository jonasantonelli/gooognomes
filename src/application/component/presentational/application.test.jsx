/**
 * @jest-environment jsdom
 */
import React from 'react';
import { shallow } from 'enzyme';
import App from './application.jsx';

describe('Application Component', () => {

  let children,
      wrapper;

  beforeEach(() => {
    children = <div className="example" />
    wrapper = shallow(<App>{ children }</App>);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });


  it('should contain children', () => {
    expect(wrapper.exists('.example')).toEqual(true);
  });

});
