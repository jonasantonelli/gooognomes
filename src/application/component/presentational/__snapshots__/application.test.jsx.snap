// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application Component should match snapshot 1`] = `
<main
  role="main"
>
  <div
    className="example"
  />
</main>
`;
